drivers/adc/ad7768-1/ad77681.h

```diff
@@ -610,7 +610,7 @@ int32_t ad77681_set_power_mode(struct ad77681_dev *dev,
 int32_t ad77681_set_mclk_div(struct ad77681_dev *dev,
 			     enum ad77681_mclk_div clk_div);
 int32_t ad77681_spi_read_adc_data(struct ad77681_dev *dev,
-				  uint8_t *adc_data
+				  uint8_t *adc_data,
```

Comment:
This fix is already merged to master:
https://github.com/analogdevicesinc/no-OS/commit/a00c63c9c05930c079afdb0b295222db9f7656dc